UI/index.html

```diff
@@ -0,0 +1,386 @@
+<!DOCTYPE html>
+<html lang = "en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="/Users/vladimirbusygin/Desktop/UP/UI/styles.css">
+    <title>YM Market</title>
+</head>
+
+<body>
+    <header class="header">
+        <div class="header_name_first">YM</div> 
```

Comment:
Dashes are most commonly used as words delimiter in HTML instead of underscores.

Reply:
Thanks, I'll spell it right.